src/main/interface/cli.c

```diff
@@ -4650,5 +4824,23 @@ void cliEnter(serialPort_t *serialPort)
 void cliInit(const serialConfig_t *serialConfig)
 {
     UNUSED(serialConfig);
+    for (uint32_t i = 0; i < valueTableEntryCount; i++) {
+        if (strcmp(valueTable[i].name, "osd_vbat_pos") == 0) {
```

Comment:
What's this code do?

Reply:
It finds the start and end index of OSD elements in the valueTable array.  These indexes are used in dumpAllValues and cliDumpOsdProfile to know when to also include the osd profile value for that OSD element.  For example, for osd_vbat_pos  the following is printed:
set osd_vbat_pos = 298
osdprofile osd_vbat_pos = 1
for other elements set {name} = {value} is printed.